src/cargo-kani/src/session.rs

```diff
@@ -222,4 +223,27 @@ impl InstallType {
             }
         }
     }
+
+    pub fn rust_toolchain(&self) -> Result<PathBuf> {
+        // This is a compile-time constant, not a dynamic lookup at runtime,
+        // so we get the toolchain we're built with (i.e. that from rust_toolchain.toml)
+        let toolchain = env!("RUSTUP_TOOLCHAIN");
```

Comment:
This is a tricky one, eh? We should be careful with custom toolchain override and users setting up this variable in the first place. That said, as long as we automate our releases to not do that, we should be OK.